### PR TITLE
Wb 13 improve docker setup

### PR DESCRIPTION
**Pull request by charlieduong94**
Quick PR to improve our docker setup

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+ENV HOME=/usr/windbreaker
+
+RUN mkdir -p $HOME
+
+WORKDIR $HOME
+
+# install git
+RUN apk update && apk upgrade && \
+  apk add --no-cache git openssh
```
Comment (Druotic) [inline Dockerfile]: Why include openssh? You can connect to docker containers without ssh. `docker exec -it <container_name> bash` (this applies to the other repos/prs as well)

File: Dockerfile_bundle [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+RUN mkdir -p $HOME
+
+WORKDIR $HOME
+
+# install git
+RUN apk update && apk upgrade && \
+  apk add --no-cache git openssh
+
+ADD . $HOME
```
Comment (Druotic) [inline Dockerfile_bundle]: Can we specify the volumes in windbreaker-deploy's docker-compose.yml file and use `ADD package.json $HOME` instead? If so, we can move the entrypoint/command to the docker-compose files and this file becomes identical to the other dockerfile. (same for other repos)

File: Dockerfile (hunk 1 above)
Comment (charlieduong94) [reply Dockerfile]: Will remove

File: Dockerfile_bundle (hunk 2 above)
Comment (charlieduong94) [reply Dockerfile_bundle]: Fixed.

**Pull request merged**